addons/website_forum/static/src/js/website_forum.js

```diff
@@ -52,10 +52,12 @@ class WebsiteForumTagsWrapper extends Component {
     }
 
     onCreateOption(string) {
-        const choice = {
-            label: string.trim(),
-            value: `_${string.trim()}`,
-        };
+        const label = string.trim();
+        const value = string
+            .split(",")
+            .map((part) => `_${part.trim()}`)
+            .join(",");
+        const choice = { label, value };
```

Comment:
```suggestion
        const choice = {
            label: string.trim(),
            value: string
                .split(",")
                .map(part => `_${part.trim()}`)
                .join(","),
        };
```
No need for an extra variable; do it in one statement.

Reply:
I made changes as suggested. Could you please take a look? 🙂